server/tests/queries.test.js

```diff
@@ -9,8 +9,16 @@ import resolvers from '../resolvers'
 import { databaseApi, searchApi } from '../endpoints'
 import { mockArticles } from './mocks'
 
+// In-memory cache to mock redis operations
```

Comment:
Wdyt? @anniefu

Reply:
Nice! JavaScript is pretty wild with this ability to just overwrite a method on an object.